library/core/src/clone.rs

```diff
@@ -200,7 +200,7 @@ mod impls {
         bool char
     }
 
-    #[unstable(feature = "never_type", issue = "35121")]
+    #[stable(feature = "never_type", since = "1.57.0")]
```

Comment:
So I admit I don't know anything, but hasn't this ship (i.e. the 1.57 release) sailed already?